tests/integration_tests/test_config_files.py

```diff
@@ -1,136 +1,136 @@
-    )
-    # this function returns when slips is done
-    os.system(command)
-
-    assert has_errors(output_dir) is False
-
-    shutil.rmtree(output_dir)
+"""
+This file tests 2 different config files other than slips' default config/slips.conf or onfig/slips.yaml
+test/test.yaml and tests/test2.yaml
+"""
+
+from slips.main import Main
+from tests.common_test_utils import (
+    is_evidence_present,
+    create_output_dir,
+    has_errors,
+    check_for_text,
+)
+from tests.module_factory import ModuleFactory
+import pytest
+from ...slips import *
+import shutil
+import os
+
+
+alerts_file = "alerts.log"
+
+
+def create_Main_instance(input_information):
+    """returns an instance of Main() class in slips.py"""
+    main = Main(testing=True)
+    main.input_information = input_information
+    main.input_type = "pcap"
+    main.line_type = False
+    return main
+
+
+@pytest.mark.parametrize(
+    "pcap_path, expected_profiles, output_dir, redis_port",
+    [
+        (
+            "dataset/test7-malicious.pcap",
+            290,
+            "test_configuration_file/",
+            6667,
+        )
+    ],
+)
+def test_conf_file(pcap_path, expected_profiles, output_dir, redis_port):
+    """
+    In this test we're using tests/test.conf
+    """
+    output_dir = create_output_dir(output_dir)
+    output_dir = create_output_dir(output_dir)
+
+    output_file = os.path.join(output_dir, "slips_output.txt")
+    command = (
+        f"./slips.py "
+        f"-t  -e 1 "
+        f"-f {pcap_path} "
+        f"-o {output_dir} "
+        f"-c /StratosphereLinuxIPS/tests/integration_tests/test.yaml "
+        f"-P {redis_port} "
+        f"> {output_file} 2>&1"
+    )
+    # this function returns when slips is done
+    os.system(command)
+
+    assert has_errors(output_dir) is False
+
+    database = ModuleFactory().create_db_manager_obj(
+        redis_port, output_dir=output_dir
+    )
+    profiles = database.get_profiles_len()
+    # expected_profiles is more than 50 because we're using direction = all
+    assert profiles > expected_profiles
+
+    log_file = os.path.join(output_dir, alerts_file)
+
+    # testing disabled_detections param in the configuration file
+    disabled_evidence = "a connection without DNS resolution"
+    assert is_evidence_present(log_file, disabled_evidence) is False
+
+    # testing time_window_width param in the configuration file
+    assert check_for_text("in the last 115740 days", output_dir) is True
+
+    # test delete_zeek_files param
+    zeek_output_dir = database.get_zeek_output_dir()[2:]
+    assert zeek_output_dir not in os.listdir()
+
+    # test store_a_copy_of_zeek_files
+    assert "zeek_files" in os.listdir(output_dir)
+
+    # test metadata_dir
+    assert "metadata" in os.listdir(output_dir)
+    metadata_path = os.path.join(output_dir, "metadata")
+    for file in ("/StratosphereLinuxIPS/config/test.yaml", "/StratosphereLinuxIPS/config/whitelist.conf", "running_slips_info.txt"):

```

Comment:
using absolute file paths will cause CI to not be able to find these files, or if another developer has slips code installed in somewhere else other than /StratosphereLinuxIPS/, the tests will fail for them too. so always use relative file paths